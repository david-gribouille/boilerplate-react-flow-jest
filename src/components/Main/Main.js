/* @flow */

import React from 'react';
import { Helmet } from 'react-helmet';

import './Main.common.scss';

const Main = () => (
  <div className="Main">
    <Helmet>
      <title>Coucou David Gribouille</title>
      <meta charSet="utf-8" />
    </Helmet>
    <div className="Main__title">Hello David!</div>
  </div>
);

export default Main;
