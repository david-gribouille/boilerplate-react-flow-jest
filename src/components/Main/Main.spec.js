import React from 'react';
import renderer from 'react-test-renderer';
import Main from './Main';

describe('Maintest Component', () => {
  it('displays correctly without props', () => {
    const component = renderer.create(<Main />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
