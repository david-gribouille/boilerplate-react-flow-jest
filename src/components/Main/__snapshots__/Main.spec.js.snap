// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maintest Component displays correctly without props 1`] = `
<div
  className="Main"
>
  <div
    className="Main__title"
  >
    Hello David!
  </div>
</div>
`;
